packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -48,7 +48,7 @@ abstract class ConstrainedLayoutBuilder<ConstraintType extends Constraints> exte
   /// Called at layout time to construct the widget tree.
   ///
   /// The builder must not return null.
-  final Widget Function(BuildContext, ConstraintType) builder;
+  final Widget Function(BuildContext context, ConstraintType constraints) builder;
```

Comment:
Actually, per our style guide this should actually be using a typedef. Something like:

```suggestion
  final ConstrainedLayoutWidgetBuilder<ConstraintType> builder;
```

with 

```dart
/// The signature of the [ConstrainedLayoutBuilder] builder function.
typedef ConstrainedLayoutWidgetBuilder<ConstraintType extends Constraints> = Widget Function(BuildContext context, ConstraintType constraints);
```

Since you're in here, would you mind cleaning this up?